src/systemd/lcdproc.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=LCDproc system status information viewer
+Documentation=man:lcdproc(8) http://www.lcdproc.org/
+
+[Service]
+User=root
+ExecStart=/usr/bin/lcdproc -f -c /etc/lcdproc.conf
```

Comment:
Please render file to the volatile location: `/run/lcdproc` so it won't be persistent.

Reply:
OK, I could do this, but the lcdproc Debian package already has /etc/LCDd.conf as well as /etc/lcdproc.conf. FWIW, the Debian package also has LCDd.service, and it runs LCDd with /etc/LCDd.conf.

So, I suppose we can follow the Debian "flavor" and leave everything in /etc or we can follow the VyOS "flavor" and render both files in /run.

Thanks for letting me know which way to go.

Reply:
The practice is to put all rendered configs in /run/$servicename, as they don't need to persist across reboot (/etc is intended for persistent configuration by the user/administrator with a few exceptions like hosts, resolv.conf). Then specify RuntimeDirectory and WorkingDirectory in the service file to point to that /run/$servicename directory.